### ISSUE TITLE
Using R to access APIs

### ISSUE THREAD

**Issue by mairindeith** [minisciencegirl/studyGroup#190]
Hello everyone!

I'm starting to learn how to use R to access APIs (APIs allow machines to access large datasets programmatically), and was wondering if anyone else is interested in learning with me. I'd be happy to lead a session if there's interest. 
Cheers!

**Comment by minisciencegirl** [comment]
Yes!! I would definitely love to learn more and I think this would be a perfect topic for the fall. Cc: @ttimbers
